tests/aws/services/lambda_/test_lambda.py

```diff
@@ -1007,6 +1041,47 @@ def test_lambda_url_headers_and_status(self, create_lambda_function, aws_client)
         assert result.text == "i'm a teapot"
         assert result.status_code == 418
 
+    @markers.aws.validated
+    def test_lambda_update_function_url_config(self, create_lambda_function, snapshot, aws_client):
+        snapshot.add_transformer(
+            snapshot.transform.key_value(
+                "FunctionUrl", "<function-url>", reference_replacement=False
+            )
+        )
+        function_name = f"test-fnurl-echo-{short_uid()}"
+
+        create_lambda_function(
+            func_name=function_name,
+            zip_file=testutil.create_zip_file(TEST_LAMBDA_URL, get_content=True),
+            runtime=Runtime.nodejs16_x,
```

Comment:
Consider adopting the latest and much more lightweight `nodejs20.x` runtime

Reply:
🤔 there is no `Runtime.nodejs20.x` defined yet in the lambda-api?

Reply:
the order is confusing 😄 
new ones are added at the bottom